test/e2e/playwright/mmi/specs/navigation.spec.ts

```diff
@@ -18,8 +18,7 @@ const supportContactUs =
 const mmiHomePage = 'https://metamask.io/institutions/';
 const privacyAndNotice = 'https://consensys.io/privacy-notice';
 const openSeaTermsOfUse = 'https://opensea.io/securityproviderterms';
-const metamaskAttributions =
-  'https://raw.githubusercontent.com/MetaMask/metamask-extension/develop/attribution.txt';
+const metamaskAttributions = `https://raw.githubusercontent.com/MetaMask/metamask-extension/v${process.env.METAMASK_VERSION}/attribution.txt`;
```

Comment:
Similarly here, I am wondering if `process.env.METAMASK_VERSION` is set to anything

Reply:
Compared to the previous unit test, this is an end-to-end test where the extension is built and installed on the browser. During the build process, the version is assigned from the manifest. I tried running the e2e tests locally; although they did not complete successfully, the extension was installed, and the wallet was recovered. However, the process stopped at the password setup, which I completed manually. When I checked, the attribution from the settings they were using the correct link.

Reply:
>During the build process, the version is assigned from the manifest

Yes, but the build process is separate from test processes. The this file is run as part of the test process, where it's not set.

Reply:
Yes, you're right. I was able to run the tests locally, and they failed. In my previous message, I mentioned that it's working because I only checked that the extension had the correct attribution link, which it does. However, the test was expecting `https://raw.githubusercontent.com/MetaMask/metamask-extension/vundefined/attribution.txt`, but it received `https://raw.githubusercontent.com/MetaMask/metamask-extension/v11.16.15-mmi.0/attribution.txt`.

Reply:
I had to remove the test because, unfortunately, the build and test processes are separate. We can't reference the version used during the build in the test. The change has been approved by @zone-live

Reply:
Hmm, we could read `package.json` from within the test, and populate it that way. But removing it sounds OK to me too.